src/loan_form/LoanForm.jsx

```diff
@@ -28,15 +66,49 @@ export default class LoanForm extends React.Component {
 
     this.setState({ loans: newLoans })
   }
+
+  @autobind
+  findLoanLength(amountOwed, interestRate, monthlyPayment) {
+    let tempAmountOwed = amountOwed
+    let monthlyInterest = tempAmountOwed * ((interestRate / 365) ** 30.417)
```

Comment:
I'm guessing `365` is like `daysInYear`, but what is `30.417`?

Also, we know that years don't always have 365 days.

Reply:
nor do months have 30 days, unfortunately. 30.417 is the average number of days in a month (on a 365 day calendar). I basically went a quarter-way to accurate

Reply:
i think it would be helfpul, for readability, to lift these into named constants or maybe even functions (then we can change how they're calculated without changing this code).

Reply:
agreed